dev_guide/managing_images.adoc

```diff
@@ -116,6 +116,37 @@ endif::[]
 that prevents the registry client from pulling from such a tag.
 ====
 
+[[tag-naming]]
+=== Tag Naming
+Prefer more generic names for tags so that image pruning can get you rid of old
+images.
+
+Images evolve over the time and tag reflects it. It always points to the latest
+image built. If there's too much information embedded in a tag name (e.g.
+`v2.0.1-may-2016`), the tag will probably point to just one revision of image
+and won't ever be updated. Using standard image pruning options, such an image
+will never be garbage collected. If tagged by name `v2.0` instead, there's much
+higher probability of more image revisions which means longer
+xref:../architecture/core_concepts/builds_and_image_streams.adoc#image-stream-tag[tag
+history] and therefore image pruner will more probably remove old and unsed
+images.
+
+Refer to xref:../admin_guide/pruning_resources.adoc#pruning-images[pruning
+images] for more information.
+
+The tag naming convention is up to you. Here's a few examples for inspiration:
+
+[width="40%",frame="topbot",options="header"]
+|======================
+|Description |Example
+|Revision    |`v2.0.1`
+|Architecture|`v2.0-x86_64`
+|Base image  |`v1.2-centos7`
+|======================
+
+If you require dates in the tag names. You should periodically inspect old and
```

Comment:
That's right. I added some more explanation but I'm not sure if it makes it more clear. It's not that easy to explain.